### ISSUE TITLE
App Repository validation failed 3.7.1

### ISSUE THREAD

**Issue by plnordquist** [kubeapps/kubeapps#1785]
### Description:

When I attempt to add a new repository to my kubeapps it says Validation Failed and claims to fail parsing the validation response.  Forcing it to install the repository works and the repository synchronizes properly.

### Steps to reproduce the issue:

1. Install Kubeapps
2. Attempt to add a repository named `ingress-nginx` with URL `https://kubernetes.github.io/ingress-nginx`
3. Kubeapps claims Validation Failed
4. Click Install Repo (force)
5. Verify that there are charts from that repository in the catalog

### Describe the results you received:

Validation fails with the content:

```
Unable to parse validation response, got: {"code":200,"message":"apiVersion: v1\nentries:\n  ingress-nginx:\n  - apiVersion: v1\n    appVersion: 0.32.0\n    created: \"2020-05-21T14:53:57.568416196Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: e4ce612f4947a0d8e196e4a030f13f4b29a9ba9b7e4d81406098f7a0b2139c7f\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.3.0/ingress-nginx-2.3.0.tgz\n    version: 2.3.0\n  - apiVersion: v1\n    appVersion: 0.32.0\n    created: \"2020-05-20T16:02:50.526507073Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: 216956498286a7a791ecf453f5dce37c00002c05df42356c4f3b48bfb1b7a990\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.2.0/ingress-nginx-2.2.0.tgz\n    version: 2.2.0\n  - apiVersion: v1\n    appVersion: 0.32.0\n    created: \"2020-05-04T20:49:33.702713789Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: 52e8f50066c6d3d81d0b9f8f5cc064f49da784344c92e109e0c0b6b0bd925c8e\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.1.0/ingress-nginx-2.1.0.tgz\n    version: 2.1.0\n  - apiVersion: v1\n    appVersion: 0.32.0\n    created: \"2020-05-01T15:17:43.751789075Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: f62f6a20858f4bf5acb68e18f31a148002fae7378b68cff7325d135a99d6c30c\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.0.3/ingress-nginx-2.0.3.tgz\n    version: 2.0.3\n  - apiVersion: v1\n    appVersion: 0.31.1\n    created: \"2020-04-30T02:45:13.213308357Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: fa43fb36ceb7ad739c1cec020c979d0b4e6f93945bcef9f6db91c7d836a1c0df\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.0.2/ingress-nginx-2.0.2.tgz\n    version: 2.0.2\n  - apiVersion: v1\n    appVersion: 0.31.1\n    created: \"2020-04-27T14:44:46.676012721Z\"\n    description: Ingress controller for Kubernetes using NGINX as a reverse proxy\n      and load balancer\n    digest: 65f3797b58e9bc94eeaa0238b8e50db0153288f6081b251e95406dfde5ab2d3f\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.0.1/ingress-nginx-2.0.1.tgz\n    version: 2.0.1\n  - apiVersion: v1\n    appVersion: 0.31.0\n    created: \"2020-04-26T21:05:22.019562847Z\"\n    description: An nginx Ingress controller that uses ConfigMap to store the nginx\n      configuration.\n    digest: 6b64d60da3b00130210b58a346c3f6a62dbebfcd3a9d8845a9137c9b0f02df1d\n    home: https://github.com/kubernetes/ingress-nginx\n    icon: https://upload.wikimedia.org/wikipedia/commons/thumb/c/c5/Nginx_logo.svg/500px-Nginx_logo.svg.png\n    keywords:\n    - ingress\n    - nginx\n    kubeVersion: '>=1.10.0-0'\n    maintainers:\n    - name: ChiefAlexander\n    name: ingress-nginx\n    sources:\n    - https://github.com/kubernetes/ingress-nginx\n    urls:\n    - https://github.com/kubernetes/ingress-nginx/releases/download/ingress-nginx-2.0.0/ingress-nginx-2.0.0.tgz\n    version: 2.0.0\ngenerated: \"2020-04-26T21:05:21.59370903Z\"\n"}
```

### Describe the results you expected:

Validation should not fail on a valid repository.

### Additional information you deem important (e.g. issue happens only occasionally):

I tried this with ingress-nginx repository from above and a `ceph-csi` repository with url `https://ceph.github.io/csi-charts` and they both failed with a json object in the output.

### Version of Helm, Kubeapps and Kubernetes:

- Output of `helm version`:

```
version.BuildInfo{Version:"v3.2.1", GitCommit:"fe51cd1e31e6a202cba7dead9552a6d418ded79a", GitTreeState:"clean", GoVersion:"go1.13.10"}
```

- Output of `helm list <kubeapps-release-name>`:

```
NAME            NAMESPACE       REVISION        UPDATED                                 STATUS          CHART           APP VERSION
kubeapps        services        19              2020-06-08 15:29:55.205611449 -0700 PDT deployed        kubeapps-3.7.1  v1.10.1
```

- Output of `kubectl version`:

```
Client Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.10", GitCommit:"f3add640dbcd4f3c33a7749f38baaac0b3fe810d", GitTreeState:"clean", BuildDate:"2020-05-20T14:00:52Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"16", GitVersion:"v1.16.10", GitCommit:"f3add640dbcd4f3c33a7749f38baaac0b3fe810d", GitTreeState:"clean", BuildDate:"2020-05-20T13:51:56Z", GoVersion:"go1.13.9", Compiler:"gc", Platform:"linux/amd64"}
```

**Comment by andresmgot** [comment]
hi @plnordquist,

It seems that somehow, even if you are using the latest version of the chart, you are using an old version of the dashboard image. Have you manually modified any of the deployments? Can you check that the deployment `kubeapps-internal-dashboard` has the image `docker.io/bitnami/kubeapps-dashboard:1.10.1-debian-10-r0`?

**Comment by plnordquist** [comment]
@andresmgot, this was a browser cache issue, I just emptied the cache for my site and hard reloaded and it started working for both chart version 3.7.0 and 3.7.1.  I ended up testing both charts because the 3.7.1 version had a different image version than the one you quoted, it was `docker.io/bitnami/kubeapps-dashboard:1.10.1-debian-10-r2`.  I'm using Chrome and when I went back to 3.6.0, force reloaded the page, and then went back to 3.7.1 it seemed to load the main index page from cache so it didn't attempt to load the new javascript and css bundles.
